tests/cephfs/cephfs_utilsV1.py

```diff
@@ -167,17 +167,83 @@ def auth_list(self, clients, **kwargs):
                 client.exec_command(
                     sudo=True, cmd=f"ceph auth del client.{client.node.hostname}"
                 )
-            client.exec_command(
-                sudo=True,
-                cmd=f"ceph auth get-or-create client.{client.node.hostname}"
-                f" mon 'allow *' mds "
-                f"'allow *, allow * path=/' osd 'allow *'"
-                f" -o /etc/ceph/ceph.client.{client.node.hostname}.keyring",
-            )
-            client.exec_command(
-                sudo=True,
-                cmd=f"chmod 644 /etc/ceph/ceph.client.{client.node.hostname}.keyring",
-            )
```

Comment:
Any other suites this code is being used?

Can we run all other suites also once?